client/src/components/game/Game.css

```diff
@@ -59,30 +59,20 @@
   text-decoration: underline;
 }
 
+.top-right {
+  display: flex;
+  flex-direction: row;
+  gap: 1.5rem;
+}
+
 .top-btns {
   display: flex;
   flex: row;
   gap: 1rem;
+  min-height: 4rem;
```

Comment:
If I didn't set `min-height` to `4em`, this is the UI change when a player joins the lobby:

https://imgur.com/4jPKDpC